docs/concepts/privacy-and-security.md

```diff
@@ -11,9 +11,9 @@ localization_priority: Normal
 
 Office Add-ins are secured by an add-in runtime environment, a multiple-tier permissions model, and performance governors. This framework protects the user's experience in the following ways: 
 
-- Access to the host application's UI frame is managed.
+- Access to the Office client application's UI frame is managed.
```

Comment:
@alison-mk, I accidentially deleted the whole thread...

You had asked something to the effect of "When do you use "Office client application" instead of "Office application"".

If the surrounding context could leave people confused about the Office Add-in versus Office application, I added "client". I'm not sure if it's ever really necessary, but adding client seemed to help clarify (to me) that its the Office program the customer is running.

We can probably work towards a more concrete rule in the future.